docs/integrations/ide/vscode.rst

```diff
@@ -26,3 +26,43 @@
 
 #. The remote host will now be available in the VS Code **Remote Explorer**. For further detail,
    please refer to the `official documentation <https://code.visualstudio.com/docs/remote/ssh>`__.
+
+############################
+ Addendum for Windows Users
+############################
```

Comment:
are these instructions really a good idea?

To me, these instructions are basically so difficult that we may as well just say "you can't run --show-ssh-command from inside wsl; that's not supported".  Having to follow these instructions for every shell a user starts is crazy.

Alternatively, we could detect if we were inside wsl in the python code:

```python
if "WSL" in os.uname().release:
    ...
```

Though I'm not sure if this affects enough users to fix it.  It probably does affect enough users that we could emit a warning when somebody asks for --show-ssh-command within WSL though, that's probably the lower-limit on what a fix should entail.

Reply:
I think there's value in having documentation for users who REALLY want to use this feature. I like adding the warning and adding a disclaimer to the docs along the lines of "we do not recommend doing this"

Reply:
I'd much prefer telling those users "install just determined in not-wsl for this feature".  That's two sentences and it's actually usable for end users.